backend/helpers/e2ehelper/data_flow_tester.go

```
@@ -119,7 +119,7 @@ func NewDataFlowTester(t *testing.T, pluginName string, pluginMeta plugin.Plugin
 
 // ImportCsvIntoRawTable imports records from specified csv file into target raw table, note that existing data would be deleted first.
 func (t *DataFlowTester) ImportCsvIntoRawTable(csvRelPath string, rawTableName string) {
-	csvIter := pluginhelper.NewCsvFileIterator(csvRelPath)
+	csvIter, _ := pluginhelper.NewCsvFileIterator(csvRelPath)
```

Do we need to care about this err?

---

As a testing helper function, it could throw panic if something went wrong. However, as a function called by the plugin, it would be better to return an error and let the caller decides how to handle it.